blocks/api/raw-handling/index.js

```diff
@@ -1,32 +1,67 @@
 /**
  * External dependencies
  */
-import { compact } from 'lodash';
-import showdown from 'showdown';
+import { find, flatMap, filter, compact } from 'lodash';
 
 /**
  * Internal dependencies
  */
-import { createBlock, getBlockTransforms, findTransform } from '../factory';
-import { getBlockType, getUnknownTypeHandlerName } from '../registration';
+import { createBlock, getBlockTransforms } from '../factory';
+import { getBlockType } from '../registration';
 import { getBlockAttributes, parseWithGrammar } from '../parser';
 import normaliseBlocks from './normalise-blocks';
-import stripAttributes from './strip-attributes';
 import specialCommentConverter from './special-comment-converter';
-import commentRemover from './comment-remover';
-import createUnwrapper from './create-unwrapper';
 import isInlineContent from './is-inline-content';
-import formattingTransformer from './formatting-transformer';
+import phrasingContentReducer from './phrasing-content-reducer';
 import msListConverter from './ms-list-converter';
 import listReducer from './list-reducer';
 import imageCorrector from './image-corrector';
 import blockquoteNormaliser from './blockquote-normaliser';
-import tableNormaliser from './table-normaliser';
-import inlineContentConverter from './inline-content-converter';
-import embeddedContentReducer from './embedded-content-reducer';
-import { deepFilterHTML, isInvalidInline, isNotWhitelisted, isPlain, isInline } from './utils';
+import figureContentReducer from './figure-content-reducer';
 import shortcodeConverter from './shortcode-converter';
-import slackMarkdownVariantCorrector from './slack-markdown-variant-corrector';
+import markdownConverter from './markdown-converter';
+import iframeRemover from './iframe-remover';
+import {
+	deepFilterHTML,
+	isPlain,
+	removeInvalidHTML,
+	getPhrasingContentSchema,
+	getBlockContentSchema,
+} from './utils';
+
+/**
+ * Browser dependencies
+ */
+const { log, warn } = window.console;
+
+export { getPhrasingContentSchema };
+
+/**
+ * Filters HTML to only contain phrasing content.
+ *
+ * @param {string} HTML The HTML to filter.
+ *
+ * @return {string} HTML only containing phrasing content.
+ */
+function filterInlineHTML( HTML ) {
+	HTML = deepFilterHTML( HTML, [ phrasingContentReducer ] );
+	HTML = removeInvalidHTML( HTML, getPhrasingContentSchema(), { inline: true } );
+
+	// Allows us to ask for this information when we get a report.
+	log( 'Processed inline HTML:\n\n', HTML );
+
+	return HTML;
+}
+
+function getRawTransformations() {
+	return filter( getBlockTransforms( 'from' ), { type: 'raw' } )
+		.map( ( transform ) => {
+			return transform.isMatch ? transform : {
+				...transform,
+				isMatch: ( node ) => transform.selector && node.matches( transform.selector ),
```

Comment:
It [seems](https://developer.mozilla.org/en-US/docs/Web/API/element/matches) like `Element#matches` isn't supported on IE. Per Slack, resorting to a lib to polyfill may be in order.

Reply:
Will adjust.

Reply:
Maybe in the block library we can just keep using `nodeName` for the simple checks and avoid pulling in the polyfill and confuse block authors.